internal/cmd/iam/command_acl_delete.go

```diff
@@ -46,11 +46,11 @@ func (c *aclCommand) delete(cmd *cobra.Command, _ []string) error {
 		return c.handleACLError(cmd, err, response)
 	}
 
-	promptMsg := errors.DeleteACLConfirmMsg
+	promptMsg := pacl.DeleteACLConfirmMsg
 	if len(bindings) > 1 {
-		promptMsg = errors.DeleteACLsConfirmMsg
+		promptMsg = pacl.DeleteACLsConfirmMsg
 	}
-	if ok, err := form.ConfirmDeletion(cmd, promptMsg, ""); err != nil || !ok {
+	if ok, err := form.ConfirmDeletionYesNoCustomPrompt(cmd, promptMsg); err != nil || !ok {
```

Comment:
The user input is `y/n` and the function takes in a prompt for the user instead of using the boilerplate prompt.

Reply:
Since we're passing `promptMsg` here, and it's clear we're passing a custom prompt message, maybe we can just do this:
```suggestion
	if ok, err := form.ConfirmDeletionYesNo(cmd, promptMsg); err != nil || !ok {
```

Reply:
That's what I'm calling the boilerplate version `ConfirmDeletionYesNo`. I'm open to naming suggestions, or other suggestions.

I guess I could just have one version that takes in a prompt, but then I would be passing in the same prompt for the majority of these calls but that just increases the amount of boilerplate code in the delete files.

Incidentally, this is why I wish golang had overloading.